nixos/modules/system/boot/systemd/coredump.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, utils, ... }:
+
+with lib;
+
+let
+  cfg = config.systemd.coredump;
+  systemd = config.systemd.package;
+in {
+  options = {
+    systemd.coredump.enable = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether core dumps should be processed by
+        <command>systemd-coredump</command>. If disabled, core dumps
+        appear in the current directory of the crashing process.
+      '';
+    };
+
+    systemd.coredump.extraConfig = mkOption {
```

Comment:
```suggestion
    extraConfig = mkOption {
```

Reply:
Same as above. This conflicts with @SuperSandro2000's comment to keep the code as-is in this PR as much as possible to help the review process.